src/render/FontHandler.cpp

```diff
@@ -96,18 +98,21 @@ void FontHandler::renderTextBox(std::string text, double x, double y, double z,
   }
 
   double off_x = (width - fontWidth) / 2.0f;
-  double off_y = (height - fontHeight) / 2.0f;  //Bounding box isn't centered, so we need to add fudge factor
+  double off_y = (height - fontHeight) / 2.0f; // Bounding box isn't centered, so we need to add fudge factor
 
-  if (alignment == TextAlignment::LEFT) off_x = TEXTBORDER;
-  if (alignment == TextAlignment::RIGHT) off_x = width - TEXTBORDER - fontWidth;
+  if (alignment == TextAlignment::LEFT)
+    off_x = TEXTBORDER;
+  if (alignment == TextAlignment::RIGHT)
+    off_x = width - TEXTBORDER - fontWidth;
```

Comment:
Add curly brackets to follow the rest of the project

Reply:
Also should this be an if/else?

Reply:
Done.